service/build.gradle

```diff
@@ -59,6 +60,17 @@ jacoco {
     applyTo jacocoBootRun
 }
 
+test {
+    systemProperties['pact.rootDir'] = "$buildDir/pacts"
+    systemProperties['pact.provider.version'] = "$project.version"
+}
+
+tasks.register("pactTests", Test) {
```

Comment:
This is all the same configuration used in [BPM](https://github.com/DataBiosphere/terra-billing-profile-manager/pull/168)